docs/swagger.yaml

```diff
@@ -61,20 +61,20 @@ paths:
           schema:
             type: "array"
             items:
-              $ref: "#/definitions/FieldError"
+              $ref: "#/definitions/Error"
         401:
           description: "Unauthorized"
         500:
           description: "Internal Server Error"
           schema:
             type: "array"
             items:
-              $ref: "#/definitions/FieldError"
+              $ref: "#/definitions/Error"
         503:
           description: "Service unavailable"
         504:
           description: "Gateway Timeout"
-  /api/v1/agreement/{agreementId}:
+  /api/v1/agreements/{agreementId}:
```

Comment:
While I agree this is how the api should be as it follows common convention, the endpoints are currently `/api/v1/agreement(/*)?`, we don't use plurals. Same goes for fetching charges, `/api/v1/charge(/*)?`.